api/app/serializers/concerns/spree/api/v2/resource_serializer_concern.rb

```diff
@@ -12,9 +12,20 @@ def self.included(base)
           # include standard attributes
           base.attributes(*model_klazz.json_api_columns)
           # include money attributes
-          base.attributes(*model_klazz.new.methods.find_all do |m|
```

Comment:
@sebastian-palma let's make a separate PR with this change

Reply:
sure @damianlegawiec, [here](https://github.com/spree/spree/pull/11523) is, I'm gonna cherry-pick those commits